src/Symfony/Component/VarDumper/Caster/GmpCaster.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+/**
+ * Transform a GMP object to a integer or a string. It need the gmp php extension.
+ *
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+class GmpCaster
+{
+    public static function castInt(\GMP $value): array
+    {
+        return array('value' => gmp_intval($value));
+    }
+
+    public static function castString(\GMP $value): array
+    {
+        return array('value' => gmp_strval($value));
```

Comment:
This means "value" will be displayed as if it were a public property on the object, which is wrong
should be a virtual property (see other casters)
Also: this will display the value as if it were a string, which is also a bad hint. I suggest to use a ConstStub to wrap the value, to that it is displayed in a more specific way. I invite you to compare with/without and decide.